themes/bootstrap3/templates/cart/form-record-hidden-inputs.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+  $recordId = $this->record?->getUniqueId() ?? $this->driver?->getUniqueId() ?? $this->recordId ?? false;
```

Comment:
What is the circumstance under which we need to support `$this->record` instead of `$this->driver`? Might it be better to remap that variable name when including this template in that context (via the parameters to renderInContext) instead of checking multiple possible variable names here? That feels like it might make the code slightly more straightforward.

Reply:
yea, it would be also good. Can tune it.

Reply:
Thanks! Just please let me know when you're ready for another review. :-)

Reply:
@demiankatz seems like it was a mixup with the this record and not required so i removed it.